<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeDisplay</class>
 <widget class="QWidget" name="EmployeeDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_view_salary">
      <attribute name="title">
       <string>查看工资</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="pg_choose_view_way">
          <widget class="QPushButton" name="pb_view_one_month_info">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>130</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>查看详细（某月）信息</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_view_multi_month_info">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>200</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>查看比较（多月）信息</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pb_choose_one_month_view">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>91</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="cb_choose_month">
             <item>
              <property name="text">
               <string>1月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11月</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12月</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="cb_choose_year">
             <item>
              <property name="text">
               <string>2016年</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2017年</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2018年</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2019年</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>92</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="7">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>91</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>92</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <widget class="QComboBox" name="cb_choose_view_way">
             <item>
              <property name="text">
               <string>表格显示</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>饼状图显示</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QPushButton" name="pb_watch">
             <property name="text">
              <string>查看</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="8">
            <widget class="QStackedWidget" name="show_way">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="table_show">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QWidget" name="table_show_widget" native="true">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QTextEdit" name="textEdit"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="pie_chart_show">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="Pie" name="w_pie_view" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pg_choose_more_month_view">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>166</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cb_year_choose">
                <item>
                 <property name="text">
                  <string>2016年</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2017年</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2018年</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2019年</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>165</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="pb_search">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>166</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0" colspan="5">
               <widget class="Histogram" name="w_his_view" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_attendance_rate">
      <attribute name="title">
       <string>查看考勤</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="attendance_choose">
          <widget class="QPushButton" name="pb_someday_view">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>130</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>查看某日考勤</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_some_month_view">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>210</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>查看某月考勤率</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pg_someday_view">
          <widget class="QComboBox" name="le_search_day">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>30</y>
             <width>69</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>31</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
          <widget class="QComboBox" name="le_search_month">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>30</y>
             <width>69</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>30</y>
             <width>41</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>年份：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>30</y>
             <width>41</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>月份：</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit_2">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>80</y>
             <width>621</width>
             <height>311</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>550</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>30</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>天：</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_search_year">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pg_some_month_view">
          <widget class="QLineEdit" name="le_view_attendance_year">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>30</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
          <widget class="QComboBox" name="le_view_attendance_month">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>30</y>
             <width>69</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>09</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>30</y>
             <width>41</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>年份：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>30</y>
             <width>41</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>月份：</string>
           </property>
          </widget>
          <widget class="Pie" name="w_employee_attendance_pie_show" native="true">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>80</y>
             <width>561</width>
             <height>331</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_competitive_analysis">
      <attribute name="title">
       <string>竞争力分析</string>
      </attribute>
      <widget class="QTextEdit" name="te_process">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>631</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lb_result">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>521</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 138, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pie</class>
   <extends>QWidget</extends>
   <header location="global">pie.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Histogram</class>
   <extends>QWidget</extends>
   <header location="global">histogram.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
